corehq/apps/app_manager/xform.py

```diff
@@ -2133,7 +2133,7 @@ def infer_vellum_type(control, bind):
 
 def find_missing_instances(form_source):
     from corehq.apps.app_manager.suite_xml.post_process.instances import get_all_instances_referenced_in_xpaths
-    instance_declaration_re = r"""<instance.*id=\"([\w\-:]+)\""""
+    instance_declaration_re = r"""<instance[^>]*id=\"([\w\-:]+)\""""
```

Comment:
I was trying to avoid parsing the form AGAIN but I guess xml parsing should be fast. Updated.